python/ctsm/subset_data.py

```diff
@@ -551,6 +551,7 @@ def subset_point(args, file_dict: dict):
         create_datm=args.create_datm,
         create_user_mods=args.create_user_mods,
         dom_pft=args.dom_pft,
+        evenly_split_cropland=args.evenly_split_cropland,
```

Comment:
@samsrabin convinced me that, just as we allow users to specify a dominant pft when running `subset_data`, we should allow them to specify `evenly_split_cropland`.

Reply:
I like this as well. I think the name of the option is nicely descriptive as well.